### ISSUE TITLE
Inconsistent Returns

### ISSUE THREAD

**Issue by paulruescher** [mweibel/facebook.ex#47]
Noticed that some methods return a tuple, like:
```
{:json, %{"email" => email}} = Facebook.me([fields: "email"], access_token)
```

Whereas other methods return a map, like: 
```
%{"access_token" => access_token} = Facebook.accessToken
```

Is this done for a particular reason? If not, would you be open to a PR that makes the returns consistent?

**Comment by tfinnell** [comment]
I'm fairly certain these inconsistencies aren't deliberate, but more a reflection of the immaturity of the project. 

The `{:json, ...}` style returns are coming from hackney. I would have loved to see `{:ok, ...}` or `{:error, ...}` when I first used this library as that seems to be a more common convention across the elixir ecosystem from what I've seen.

There's definitely value in normalizing these outputs to simplify ease of use for users and providing 
a convention for contributors to follow.

**Comment by paulruescher** [comment]
That's what I was thinking as well: `{:ok, resp} / {:error, reason}`.

If there's no other feedback, I'll try and start working on a PR this week that'll address standardized returns. I'll need to spend a bit of time trying to grok the way returns exist

**Comment by mweibel** [comment]
That would be awesome @paulruescher!

**Comment by paulruescher** [comment]
OK, easy question: I have some tests failing due to the given @testPageId. I'm assuming that's somehow attached to an app ID that I'm not associated with. Am I correct? If yes, can you maybe send me your `.env` file (or related)?

**Comment by mweibel** [comment]
yeah I still wonder how to do that across PRs. Issue is that I need to provide app/page id and API key in the travis-CI run. They're encrypted and I don't really want to give them to all people ;)
If you have an idea how to improve that.. let me know. I'll probably switch to recorded responses instead (mocking away the FB API completely). But I didn't have time yet to do that. 
Until then: is it possible for you to create an own app which you can test with?

**Comment by paulruescher** [comment]
Yeah, I'll give that a shot. TBH, I might just mock out responses if you're OK with that too.

**Comment by mweibel** [comment]
Sure, please do so 👍

**Comment by paulruescher** [comment]
Great. I made some decent progress tonight, so I'll probably have a WIP PR sometime in the next couple days. I'll keep you posted.

**Comment by paulruescher** [comment]
I thought of something. The errors that are failing are related to the test page. Would it be possible to make the test page public? It may be that the requests are just trying to hit a private page.

**Comment by mweibel** [comment]
ok, just published it:
https://www.facebook.com/Test-app-for-elixir-library-1568088063205571

**Comment by paulruescher** [comment]
Closed by #49
